### ISSUE TITLE
Cannot Delete Table With ExcelTableCollection.Delete()

### ISSUE THREAD

**Issue by jinunpark-nc** [EPPlusSoftware/EPPlus#199]
I need to delete all tables on excel file. I tried with this code, and table remains.

Test_Style.xlsx has tables.

```cs
        [Test]
        public void DeleteTable()
        {
            var originalPath = "Test_Style.xlsx";

            var resultPath = "result.xlsx";

            using var originalExcelPackage = new ExcelPackage(new FileInfo(originalPath));

            foreach (var worksheet in originalExcelPackage.Workbook.Worksheets)
            {
                while (worksheet.Tables.Count > 0)
                {
                    worksheet.Tables.Delete(0);
                }
            }

            originalExcelPackage.SaveAs(new FileInfo(resultPath));

            using var newPackage = new ExcelPackage(new FileInfo(resultPath));

            foreach (var worksheet in newPackage.Workbook.Worksheets)
            {
                Assert.That(worksheet.Tables.Count, Is.EqualTo(0));
            }
        }
```

And I discovered ExcelTableCollection.Delete() and found it deletes on parsed instance but does not delete actual xml nodes.

This link shows how to delete table on worksheet. But it does not delete table xml, just remove link to table.

https://stackoverflow.com/questions/36359047/unable-to-delete-table-using-epplus

**Comment by JanKallman** [comment]
This is a bug. I will supply a fix shortly.

**Comment by jinunpark-nc** [comment]
Thank you!

**Comment by JanKallman** [comment]
Fixed in 5.4.1
